core/src/main/java/com/amplifyframework/api/graphql/Operation.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.api.graphql;
+
+/**
+ * Represents a GraphQL operation.
+ */
+public enum Operation {
```

Comment:
Are these named in reverse? Should an `Operation` have a `getOperationType()`, which returns an `OperationType.SUBSCRIPTION`, etc.?

Reply:
I went back and forth on this as well, and can't remember why I landed where I did.  Thinking about it again now though, I think flipping them makes sense.  Will change.